desk/src/components/FormControls/Link.vue

```diff
@@ -0,0 +1,100 @@
+<template>
+	<div class="flex flex-col gap-1.5">
+		<span v-if="props.label" class="block text-xs text-gray-600">
+			{{ props.label }}
+		</span>
+		<Autocomplete
+			ref="autocompleteRef"
+			size="sm"
+			v-model="value"
+			:placeholder="`Select ${doctype}`"
+			:options="options.data"
+			:label="label"
+		/>
+	</div>
+</template>
+
+<script setup>
+import { createResource, Autocomplete, debounce } from "frappe-ui"
+import { ref, computed, watch } from "vue"
+
+const props = defineProps({
+	doctype: {
+		type: String,
+		required: true,
+	},
+	modelValue: {
+		type: String,
+		required: true,
+		default: "",
+	},
+	label: {
+		type: String,
+		required: false,
+	},
+	filters: {
+		type: Object,
+		default: {},
+	},
+})
+
+const emit = defineEmits(["update:modelValue"])
```

Comment:
@ruchamahabal just a suggestion: if you can upgrade to Vue 3.4+, the sync setup here can be simplified with [`defineModel`](https://vuejs.org/api/sfc-script-setup.html#definemodel)

Reply:
Yes, I have already used defineModel in the other components 

https://github.com/frappe/frappe/pull/26727/files#diff-6d43d216067afee155e9ede2db9aeb1e9b9a85a9a217e99eeac59decea5fc7d7R125

I created the Link component while writing the `hrms` PWA (before defineModel was supported) and used the same component here. Will refactor in my next PR